src/mpl2/src/rtl_mp.cpp

```diff
@@ -331,10 +334,10 @@ bool rtl_macro_placer(const char* config_file,
         string orientation = macros[j].GetOrientation();
         float ux = lx + width;
         float uy = ly + height;
-        lx = round(lx / pitch_x) * pitch_x;
-        ux = round(ux / pitch_x) * pitch_x;
-        ly = round(ly / pitch_y) * pitch_y;
-        uy = round(uy / pitch_y) * pitch_y;
+        lx = ceil(lx / pitch_x) * pitch_x;
```

Comment:
do you want floor for lx/ly?

Reply:
If it is OK I want revisit this and the above comment in a sep PR later this week, I am going to back out this change for now